Sources/MapboxNavigation/LaneView.swift

```diff
@@ -71,10 +83,10 @@ extension LaneIndication {
             secondaryIndication = .straightAhead
         } else if !primaryIndication.isDisjoint(with: .rights) && indications.contains(.slightLeft) {
             // No assets for slight or sharp opposite turns.
-            secondaryIndication = .slightLeft
+            secondaryIndication = .left
```

Comment:
We still don’t have assets for slight or sharp opposite turns, but at least we have assets for opposite turns. A user is more likely to distinguish between a left lane and a left-right lane than between a left lane and a slight left lane, even if the maneuver requires going slight left.